vela-templates/definitions/internal/policy/topology.cue

```diff
@@ -0,0 +1,20 @@
+"topology": {
+	annotations: {}
+	description: "Determining the destination where components should be deployed to."
+	labels: {}
+	attributes: {}
+	type: "policy"
+}
+
+template: {
+	parameter: {
+		// +usage=Specify the names of the clusters to select.
+		cluster?: [...string]
+		// +usage=Specify the label selector for clusters
+		clusterLabelSelector?: [string]: string
+		// +usage=Deprecated: Use clusterLabelSelector instead.
+		deprecatedClusterSelector?: [string]: string
```

Comment:
deprecatedClusterSelector -> clusterSelector

Reply:
ok, I will modify it